.gitignore

```diff
@@ -1,3 +1,5 @@
 /build
 /cmake-build-*
 /.idea
+Cargo.lock
+/target
```

Comment:
Do you need this top level?

Reply:
If we have the top level `Cargo.toml` then unfortunately it is needed. The reason for having the top level one is the ability to run `cargo build` on the top level.